src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/reported/ReportsControllerTest.kt

```diff
@@ -69,7 +69,7 @@ class ReportsControllerTest : TestControllerBase() {
     }
```

Comment:
i think most of this controller will be VIEW role too, maybe some specific ALO ones but unlikely

Reply:
indeed - see https://github.com/ministryofjustice/hmpps-manage-adjudications-api/pull/467#discussion_r1286774976